frontend/src/components/BaseDashboard/index.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { Header as TFHeader } from "@code4ro/taskforce-fe-components";
```

Comment:
```suggestion
import { Header } from "@code4ro/taskforce-fe-components";
```
We don't have any naming conflicts here with another Header component